runtime/appruntime/metrics/gcp_cloud_monitoring_exporter.go

```diff
@@ -11,10 +11,14 @@ func init() {
 	registerProvider(providerDesc{
```

Comment:
Would it be cleaner to treat Encore Cloud as a separate provider, instead of overloading it? Not sure, up to you